webrtc.html

```diff
@@ -1897,8 +1901,8 @@ <h4>Set the RTCSessionDescription</h4>
                   <li>
                     <p>For each <code><a>RTCTrackEvent</a></code>
                     <var>trackEvent</var> in <var>trackEvents</var>,
-                    <a>fire a track event</a> named <code title=
-                    "event-track"><a>track</a></code> with <var>trackEvent</var>
+                    <a>fire an event</a> named <code title=
+                    "event-track"><a>track</a></code> using <var>trackEvent</var>
                     at the <var>connection</var> object.</p>
```

Comment:
This also seems weird, since it looks like _trackEvents_ would hold instances, whereas "fire an event" wants a class. (I haven't checked it out in context though.)